develfood_III/src/screens/SignIn/index.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { useEffect, useRef, useState } from 'react'
+import { useTheme } from 'styled-components'
+import { Alert, Animated, Image, Keyboard, StatusBar, Text, TouchableOpacity, TouchableWithoutFeedback } from 'react-native'
+import { 
+  Container, 
+  Content, 
+  Form, 
+  ForgotPassword, 
+  CreateAccountButton, 
+  ContentImage 
+} from './styles'
+import { Button } from '../../components/Button'
+import { Input } from '../../components/Input'
+import { Label } from '../../components/Label'
+import { PasswordInput } from '../../components/PasswordInput'
+import Logo from '../../assets/logo.png';
+import Image2 from '../../assets/img.png';
+import burguer from '../../assets/burguer.png';
+import pizza from '../../assets/pizza.png';
+import { isEmail } from '../../utils'
+
+export function SignIn() {
+  const [password, setPassword] = useState('');
+  const [email, setEmail] = useState('');
+  const [loading, setLoading] = useState(false);
+
+  const animated = useRef(new Animated.Value(0)).current;
+  const theme = useTheme();
+
+  async function handleSignIn() {
+      setLoading(true)
+      const emailValido = 'email@teste.com'
+      const senhaValida = '123456'
+      setTimeout(() => {
+        if (!isEmail(email)) {
+          Alert.alert('Email invalido!')
+        } else {
+          if (emailValido === email && senhaValida === password) {
+            Alert.alert('Login realizado!')
+          } else {
+            Alert.alert('Senha invalida ou Email invalido!')
+          }
+        }
+        setLoading(false)
+      }, 3000);
+      // navigation.navigate("Home");
+  }
+
+  useEffect(() => {
+    const keyboardShow = Keyboard.addListener('keyboardDidShow', () => {
+      Animated.timing(animated, {
+        toValue: 0,
+        duration: 150,
+        useNativeDriver: true
+      }).start();
+    })
+
+    const keyboadHide = Keyboard.addListener('keyboardDidHide', () => {
+      Animated.timing(animated, {
+        toValue: 10,
+        duration: 150,
+        useNativeDriver: true
+      }).start();
+    })
+
+    return () => {
+      keyboardShow.remove(),
+      keyboadHide.remove()
+    }
+  }, [])
+
+  return (
+    <>
+      <TouchableWithoutFeedback onPress={Keyboard.dismiss}>
+          <Container>
+            <Animated.View style={{width: '100%', transform: [ { translateY: animated} ] }}>
```

Comment:
Não precisa dessa logica para o iOS
Certificar que apenas será usado no Android 
Pode user o Platform e verifica se for Android faz o transform se não, não faz